components/cost-management/policies/.chainsaw-test/chainsaw-test.yaml

```
@@ -0,0 +1,136 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: label-propagation-valid-cost-center
+spec:
+  concurrent: true
```

as the the ClusterPolicy is installed as part of each test, we can not run them concurrently. This change applies also the other tests.

```suggestion
  concurrent: false
```

---

I have an interesting observation, changing this to false results in a failure for the third use case, `label-no-cost-center` . Do you know why that could be the case?
It ends up having an empty label: {}

---

try applying this change to all the `concurrent: true` occurrences in the file. Probably a task from a concurrent test is changing something shared, like deleting the ClusterPolicy.

---

You are right, but I never figured out what was causing. But tested locally after making few changes and it works